src/devices/lcec_omrg5.c

```diff
@@ -52,8 +53,36 @@ static lcec_typelist_t types[] = {
     {"OmrG5_KN150F", LCEC_OMRON_VID, 0x0000005F, 0, NULL, lcec_omrg5_init},
     {NULL},
 };
+ADD_TYPES(old_types);
```

Comment:
[There's one more issue lurking here, but it's not part of your code -- I *suspect* that `lcec_configgen` will prefer `OmrG5 names in at least *some* cases, possibly due to the order that they appear in code and possibly just due to whichever hash order Go is using for that specific run.

Similar patterns show up elsewhere.  We really should have a generic deprecation mechanism here, possibly with a "use this instead" mechanism as well, but it'll take me a while to get to it.  Kind of underwater at work again.

Reply:
So maybe I just change the order, first the new types, then the old ones!?

For the generic deprecation mechanism, maybe we add a flag for every entry of a type that indicates a deprecated type. The only issue is how to point to the new one? There's not always a direct replcement for a deprecated type, right?